### DISCUSSION TITLE
Reflect Admin dashboard actions in UI

### DISCUSSION THREAD

**Discussion by jimafisk** [pocketbase/pocketbase#266] [Q&A] [answered]
If I perform an action in the PB Admin dashboard, I'm not sure how to watch for that change via the js-sdk. For example, if I delete a user in the Admin dashboard, my custom app still has a `pocketbase_auth` key in localstorage with the original `model` information that no longer exists. If I logout via `client.AuthStore.clear()` it updates.

This would be helpful for user onboarding where you want to show different screens based on whether the user has verified their email (`user.verified`) or completed certain profile information (`user.profile.name`). Currently I have to logout and log back in to see the change reflected. 

I'm currently using `client.authStore.onChange()` but that only seems work when I call methods from my app (like `client.Users.authViaEmail()`). This could very well be a problem with how I'm structuring my app, I appreciate the help! Thank you!

**Comment by ganigeorgiev** [comment, marked as answer]
The realtime subscriptions are supported only for the collection records (including the user `profiles` collection).

If your user model has changed or has been deleted after authentication, then there is currently no easy way to get the changes without sending a `refresh()` call (PocketBase is completely stateless and doesn't have sessions, even the tokens are not stored in the db).

`client.authStore.onChange()` is triggered only when your local data changes (eg. on `client.authStore.clear()` or `client.authStore.save()`, which is called automatically on authentication, refresh, password confirmation, email verification, etc.).

If you want your clients to have always an up-to-date state of their user model you could call `client.users.refresh()` on each page reload.

In future versions, I may consider extending the realtime subscriptions and add support for changes related to the authenticated user model.

**Reply by jimafisk** [reply]
It looks like `client.users.refresh()` gets me what I needed, sorry I must have missed that in the docs. Thank you!!
